### PR TITLE
uprading UI of request_community_form

### LINKED ISSUE #386: Upgrading UI of communities page
## Description
As a [USER],
Uprage the UI of communities and meetup page

## Mocks
Here is what the current UI as of 05/18/2018 shows:
screenshot from 2018-05-19 23-23-55
Here is what the UI will look like after the upgrade:
screenshot from 2018-05-20 23-43-52

### Update [Required]
- [ ] [should meet the branding guidelines]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[9-10] hours

### PR DESCRIPTION
**Pull request by abhi20161997**
### Description
upgraded UI of request community form as per the branding

Fixes #386 

### Mocks
Before upgradation it looked like:
screenshot from 2018-05-19 23-23-55
screenshot from 2018-05-22 01-11-19

After upgradation it looks like:
screenshot from 2018-05-20 23-43-52
screenshot from 2018-05-22 01-12-35

### Type of Change:
**Delete irrelevant options.**

- Code
- User Interface

**Code/Quality Assurance Only**

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by ExactlyMay** [comment]
@abhi20161997 Can you include a screenshot?

File: systers_portal/templates/community/request_community.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <div class="mt20 mb40"></div>
   <div class="row ml20">
     <div class="col-md-12">
-      <h1>Request a new community</h1>
+      <h1 class="request_form-head">Request a new community</h1>
```
Comment (tonythomas01) [inline systers_portal/templates/community/request_community.html]: why do we have this strange casing ? ` request-form-head` please.

File: systers_portal/templates/community/request_community.html [hunk 2]
```diff
@@ -20,7 +20,7 @@ <h1>Request a new community</h1>
     </div>
     {% if community_requests %}
       <div class="col-md-4">
-        <h4> Your Community requests </h4>
+        <h4 class="request_form-head"> Your Community requests </h4>
```
Comment (tonythomas01) [inline systers_portal/templates/community/request_community.html]: same as above.

File: systers_portal/templates/community/request_community.html (hunk 1 above)
Comment (abhi20161997) [reply systers_portal/templates/community/request_community.html]: @tonythomas01 that was to selectively edit the headings under h1

**Comment by abhi20161997** [comment]
@ExactlyMay  Added the mocks.

File: systers_portal/templates/community/request_community.html (hunk 1 above)
Comment (tonythomas01) [reply systers_portal/templates/community/request_community.html]: to selectively edit, why should the casing of class attr matter ?

Comment (sa7i) [reply systers_portal/templates/community/request_community.html]: @abhi20161997 I agree with @tonythomas01 , please change it to `request-form-head`

**Comment by monajalal** [comment]
Please also attach a screenshot of what it looked before you apply changes. Also, if you have completed the tasks in the checkboxes, please check them off.

**Comment by abhi20161997** [comment]
@monajalal implemented your suggested changes.

**Comment by monajalal** [comment]
Can you also show a screenshot of the most current dropdown menu for the community in the post?

**Comment by abhi20161997** [comment]
@monajalal as I mentioned in the slack channel , the dropdown's color can't be changed. So, it's same as the old one(the one reflected in Poojitha's comment).

**Comment by monajalal** [comment]
@abhi20161997 can you please still show the dropdown menu for before and after? Just for sanity. In case, someone can change it in the future, they can look at these screenshots. Thanks.

**Comment by abhi20161997** [comment]
@monajalal uploaded

**Comment by tonythomas01** [comment]
LGTM +1

**Pull request merged**